app/models/user.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+class User extends AppModel
+{
+    public function getUserDetail($id)
+    {
+        $db = DB::conn();
+        $row = $db->row('SELECT * FROM user WHERE id = ?', array($id));
+        return $row;
+    }
+    public function getUserName($user_id)
+    {
+        $db = DB::conn();
+        $row = $db->row('SELECT username FROM user WHERE id = ?', array($user_id));
+        return $row['username'];
```

Comment:
Is this correct?

return !empty($row['username']) ? $row['username'] : false;

also I already added the exception in here